package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "add-contributor": "kcd-scripts contributors add",
     "build": "kcd-scripts build --bundle --p-react",
+    "build:react-native": "BUILD_REACT_NATIVE=true BUILD_FILENAME_PREFIX=react-native kcd-scripts build --bundle",
```

Comment:
I'm not sure the best way to combine this build step with the normal `build` without clobbering some of the niceties of `kcd-scripts build` (by using `&&` to run both commands).

Reply:
I'd just have `build:web` and a `build:native` scripts and then a `build` script that just runs them both: `npm run build:web --silent && npm run build:native --silent` 👌